### PR TITLE
Add one line for mac silicon M1 processor arm64

### PR DESCRIPTION
**Pull request by Boyquotes**

### CODE REVIEW

File: godot-solana-sdk.gdextension [hunk 1]
```diff
@@ -13,6 +13,7 @@ windows.template_release.x86_64="libgodot-solana-sdk.windows.template_release.x8
 windows.template_debug.x86_64="libgodot-solana-sdk.windows.template_debug.x86_64.so"
 macos.template_release="libgodot-solana-sdk.macos.template_release.framework"
 macos.template_debug="libgodot-solana-sdk.macos.template_debug.framework"
+macos.arm64="libgodot-solana-sdk.macos.template_debug.framework"
```
Comment (Virus-Axel) [inline godot-solana-sdk.gdextension:16]: Does it work with macos.template_debug.arm64="libgodot-solana-sdk.macos.template_debug.framework"?

Comment (Virus-Axel) [reply godot-solana-sdk.gdextension:16]: Then we can add `macos.template_release.arm64="libgodot-solana-sdk.macos.template_release.framework"`

Comment (Virus-Axel) [reply godot-solana-sdk.gdextension:16]: Otherwise the exported game might have debug symbols included

Comment (Boyquotes) [reply godot-solana-sdk.gdextension:16]: I don't know why but only macos.arm64= works...
how to make difference for release and debug lib, I don't see in the code/doc...

Comment (Virus-Axel) [reply godot-solana-sdk.gdextension:16]: I see, then we keep these changes for now. It should not matter too much if debug symbols are present in exports. Again, thanks for the contribution.

**Review by Virus-Axel — APPROVED** [review]

**Pull request merged**